scripts/restore_anonymised_db.sh

```diff
@@ -13,3 +13,11 @@ kubectl -n laa-apply-for-legalaid-uat cp ./tmp/production.anon.sql -c web laa-ap
 
 echo "Connect to the pod and run the rake task"
 kubectl -n laa-apply-for-legalaid-uat -c web exec "$POD" -- rake db:import_to_uat
+
+if [ $? -eq 0 ]; then
+   echo 'Restore complete'
+   echo "Delete the restore file from the pod"
+   kubectl -n laa-apply-for-legalaid-uat -c web exec "$POD" -- sh -c 'rm tmp/anonymised_db.sql'
```

Comment:
Do you need the second `-c` in front of the command?

Reply:
Yes it is required. It isn't the container flag that is used in kubectl, it is a flag passed to the shell to indicate that what follows is a command. At least that is my understanding but I cannot find any documentation to support it.